engine/hatchery/local/local.go

```diff
@@ -135,12 +132,27 @@ func (h *HatcheryLocal) Serve(ctx context.Context) error {
 		return fmt.Errorf("Cannot check local capabilities: %v", err)
 	}
 
+	h.BasedirDedicated = fmt.Sprintf("%s/%s", strings.TrimRight(strings.TrimSpace(h.Config.Basedir), "/"), h.Name)
```

Comment:
could you use https://golang.org/pkg/path/filepath/#Dir ?

Reply:
yep, done